airbyte-cdk/python/airbyte_cdk/destinations/vector_db_based/config.py

```diff
@@ -85,6 +85,7 @@ class ProcessingConfigModel(BaseModel):
         ...,
         title="Chunk size",
         maximum=8191,
+        minimum=1,
```

Comment:
Should I see this field being used somewhere?

Reply:
No, it's really just updating the model - once this version of the CDK is used in connectors, it will change the spec of the connector which is the actual change